ihmc-footstep-planning/src/test/java/us/ihmc/footstepPlanning/log/FootstepPlannerLoggerTest.java

```diff
@@ -74,57 +104,87 @@ public void testFootstepLoggingSpeed()
 
       long timeInMilliseconds = (long) (timeTaken * 1000);
       // This number is arbitrary, just want to make sure things aren't slow when logging, important if re-planning in a loop
-      long timeExpected = 150;
+      long timeExpected = 180;
 
       // Logging the footstep plans should not take to long, the timeExpected is arbitrary, but it ensures that the logging doesn't take too long
       assertTrue(timeInMilliseconds < timeExpected, "Time taken was: " + timeInMilliseconds + ", and the time expected was " + timeExpected);
    }
 
+   /**
+    * This test is making sure that all the {@link us.ihmc.tools.property.StoredPropertySet} parameters are getting logged correctly.
+    * We want to make sure the parameters we are using in the planning module are getting logged.
+    */
    @Test
-   public void testLogger()
+   public void testLoggedPacketsMatchThePacketsSentToThePlanner()
```

Comment:
This is awesome! They all match and added the body path planning parameters to the test. When I say "they all match" I'm refering to the stored property sets that get used in the planner. And that they match the values that are getting logged